.github/workflows/build.yml

```diff
@@ -127,6 +127,8 @@ jobs:
           gradle-cache-read-only: ${{ !inputs.is-trunk }}
           gradle-cache-write-only: ${{ inputs.is-trunk }}
           develocity-access-key: ${{ secrets.DEVELOCITY_ACCESS_KEY }}
+      - name: Verify license file
+        run: python committer-tools/verify_license.py
```

Comment:
the script needs to run the `clean releaseTarGz`, so it will rebuild whole project and it seems to me it is a bit expensive as CI. Maybe we can do more works on this?

1. run this check after `Check generated documentation`
2. `Compile and validate` should build `releaseTarGz` too
3. `verify_license.py` should be able to skip  `releaseTarGz`